apps/api/src/app/shared/shared.module.ts

```diff
@@ -78,15 +78,17 @@ const dalService = new DalService();
 
 const inMemoryProviderService = {
   provide: InMemoryProviderService,
-  useFactory: () => {
-    return new InMemoryProviderService();
+  useFactory: (enableAutoPipelining?: boolean) => {
+    return new InMemoryProviderService(enableAutoPipelining);
```

Comment:
@p-fernandez maybe worth doing an object here? In case we will want to control other aspects in the future. Wdyt?

Reply:
Is this going to be permanent? I was expecting this to be temporary and we would enable it for Cache and DistributedLock after testing for Cache.

Reply:
I think it could become, so we can reuse this service in the future with different hosts and etc... Not a must, but was just thinkinof this for the future. You can leave it as sis for now

Reply:
Let's try it quick and then I can clean it if it works for our purpose and we decide it will stay.